OracleTuxedo/core/dockerfiles/12.1.3/Dockerfile

```diff
@@ -7,25 +7,25 @@
 #
 
 # Pull base image
-FROM oraclelinux
+FROM container-registry.oracle.com/java/serverjre
 
 MAINTAINER Todd Little <todd.little@oracle.com>
```

Comment:
Is Todd Little still the maintainer? He's not the one submitting this update. If the maintainer has changed, please reflect that here.